### ISSUE TITLE
Doesn't properly track progress if using simplelog addon

### ISSUE THREAD

**Issue by ShiyoKozuki** [onimitch/ffxi-trainmon#1]
![image](https://github.com/onimitch/ffxi-trainmon/assets/58101246/2627bff8-a1ea-4a6b-8eca-a893179be7d6)

**Comment by ShiyoKozuki** [comment]
Is it because I'm using SimpleLog?

**Comment by onimitch** [comment]
Yeah I think Simplelog is the issue. Trainmon relies on the specific format of messages, but I might be able to fix it by having trainmon ignored previous modifications. 

Before that one possible thing to try is loading trainmon before simplelog in your default scripts txt. It might fix it

**Comment by ShiyoKozuki** [comment]
> Yeah I think Simplelog is the issue. Trainmon relies on the specific format of messages, but I might be able to fix it by having trainmon ignored previous modifications.
> 
> Before that one possible thing to try is loading trainmon before simplelog in your default scripts txt. It might fix it

This didn't fix it for me.

**Comment by onimitch** [comment]
Was worth a try, thanks for giving it a go. 

I checked the code and it doesn't look like it's affected by other addons, I always use the original unmodified messages. Sadly that simply means without editing the message rules for the addon to cater for your private server, this addon won't work.

**Comment by Razimus** [comment]
I'm not managing to get this addon to work on retail while using simplelogs. Not sure if this is something you plan to address or not. Didn't want to make my own issue post so I posted here.

**Comment by onimitch** [comment]
@Razimus I don't have any plans to make this addon compatible with Simplelogs at the moment.

**Comment by ShiyoKozuki** [comment]
> @Razimus I don't have any plans to make this addon compatible with Simplelogs at the moment.

Youd need to recode it to detect via packets and not text in the log

**Comment by onimitch** [comment]
@ShiyoKozuki yes that's correct
